src/pages/ExercisesEight.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect, useState } from 'react'
+import data from "../mocks/data.json"
+import Nav from "../components/Nav"
+
+export default function ExercisesEight() {
+  const [cardsData, setCardsData] = useState(data.content)
+  useEffect(() => {
+    console.log(data)
+  },[])
```

Comment:
why do you need to console here? I think it should be removed

Reply:
Seems like I forgot to remove